team_project_2.md

```diff
@@ -4,96 +4,99 @@
 
 Working together in your project team, you'll continue to apply your newly acquired skills and revisit your program from Part 1. Data Science Certificate participants will collaborate to creatively visualize their data. Participants working toward the Machine Learning Software Foundations Certificate will work together to deploy a machine learning model. For example, your team might choose to develop a sentiment analysis model for social media comments in order to enable businesses to gauge public opinion effectively.
 
-At the end of the module, all team members are encouraged to fork the repo onto their profile so that prospective employers can view the project on all of your profiles. 
+At the end of the module, all team members are encouraged to fork the repo onto their profile so that prospective employers can view the project on all of your profiles.
 
 _Reminder: Please add in the additional file structures found on the README.md._
 
 This project applies skills from the following previous modules:
 
-* Introduction to Building Software (Shell, Git, Python)
-* SQL
-* Applying Statistical Concepts (Linear regression, classification, and resampling
-* Scaling to Production
-* Visualization (Data Science Certificate)
-* Sampling (Data Science Certificate)
-* Algorithm & Data Structures (Machine Learning Software Foundations Certificate)
-* Deep Learning (Machine Learning Software Foundations Certificate)
+- Introduction to Building Software (Shell, Git, Python)
```

Comment:
These changes happened in VS code automatically. I think can be ignored, as the content hasn't changed apart from my link